### ISSUE TITLE
Allow interception of a single target desktop app

### ISSUE THREAD

**Issue by pimterry** [httptoolkit/httptoolkit#518]
This is sort-of a variation on #8 - allowing for system-wide interception, but focused on a single app.

We should be able to do this in general by:

* Redirecting network traffic via Frida (or something very similar) to capture just that one app
* Trusting our CA certificate in the app, either using the global system certificate store (as for #8) or using some Frida trickery (which could be quite challenging!)

---

_Does this affect you too? Click below and add a :+1: to vote for this and help decide where HTTP Toolkit goes next, or [go vote on the other most popular ideas so far](https://github.com/httptoolkit/feedback/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)._

**Comment by arslanameen227** [comment]
Yes, will be very helpful ietercepting single desktop application instead of whole system.  Or add a feature (if possible) to to detect traffic coming from a specific folder.
